playbooks/openshift/roles/common/handlers/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+- name: reload firewalld
```

Comment:
Funny you should ask this here. I had a hell of a time getting the handler to work with external roles, using meta dependencies. According to the documentation, that should work, but it doesn't seem to work. 

I moved it into the nginx role as well, which is where I thought it would fit best. Unfortunately, the same meta dependencies structure was in play there and caused the same problem. 

This handler is just a leftover artifact of trying to figure out how to get the nginx role working properly as a dependency of other roles. If you have ideas here, I'd love to try them out, but I'm currently planning on filing an issue on the ansible github regarding this issue.